.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give detailed descriptions of each PR, highlighting the purpose, changes made, and impact.

In a separate sentence, give me the assignees usernames excluding jjw24 and sorted in case insensitive alphabetical order.

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result separately give detailed descriptions of each PR, highlighting the purpose, changes made, and impact.

In a separate sentence, give me the assignees usernames excluding jjw24 and sorted in case insensitive alphabetical order.